### DISCUSSION TITLE
side bar navigation tooltip does not look good

### DISCUSSION THREAD

**Discussion by KassemHachi** [filamentphp/filament#6369] [General]
Hi every one , 
I have an issue , when i hover the navigation it appears like this , i want to design the tooltip of sidebar navigation 

![Screenshot_20230501_112113](https://user-images.githubusercontent.com/30910460/235440279-a659d36e-2419-4fd8-ae3a-4067594afbec.png)

**Comment by danharrin** [comment]
You are missing the `tippy.js/dist/tippy.css` import in your theme CSS file.
